model/src/test/kotlin/licenses/LicenseInfoResolverTest.kt

```diff
@@ -589,6 +610,22 @@ class LicenseInfoResolverTest : WordSpec({
             result should containCopyrightStatementsForLicenseExactly("LicenseRef-a")
             result should containCopyrightStatementsForLicenseExactly("LicenseRef-b")
         }
+
+        "!not resolve copyrights from authors in concluded license if disabled" {
+            val licenseInfos = listOf(
+                createLicenseInfo(
+                    id = pkgId,
+                    authors = authorsWithCopyright,
+                    declaredLicenses = setOf("MIT"),
+                    concludedLicense = SpdxExpression.parse("BSD-2-Clause")
+                )
+            )
+
+            val resolver = createResolver(licenseInfos, addAuthorsToCopyrights = false)
+
+            val result = resolver.resolveLicenseInfo(pkgId)
+            result should containCopyrightStatementsForLicenseExactly("BSD-2-Clause")
+        }
```

Comment:
These new tests should be in a separate commit, they are not related to the extraction of the new function.

Reply:
And please use the more explicit `.config(enabled = false)` syntax.

Reply:
Or at least explain the syntax, which is uncommon in our code base, in the commit that adds the tests.

Reply:
I moved the tests into a separate commit. Enabled.